lib/hpke/test/userinfo_vc.cpp

```diff
@@ -175,7 +175,8 @@ TEST_CASE("UserInfoClaims Field Parsing")
     { "updated_at", 42 }
   };
 
-  const auto userinfo_claims = UserInfoClaims::from_json(credentialSubject);
+  const auto userinfo_claims =
+    UserInfoClaims::from_json(credentialSubject.dump());
```

Comment:
Same raw string trick here.  (Raw strings can span multiple lines.)

Reply:
I am using the original json as the known answer for the  tests.  The tests in that section seem to only test that the strings get shuffled around correctly.